syntax = "proto3";

package api.user.request;

message RevokeDevice {

}
